oci.py

```diff
@@ -0,0 +1,67 @@
+import argparse
+import logging
+
+from oci import config
+from oci import gerrit
+from oci import jenkins
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description='A command line tool for working with Ovirt CI')
+
+    parser.add_argument(
+        '--verbose',
+        help="increase output verbosity",
+        action="store_true")
+    subparsers = parser.add_subparsers(title="commands")
+
+    build_artifacts = subparsers.add_parser(
+        "build-artifacts",
+        help="start the build-artifacts stage")
+    build_artifacts.set_defaults(command=run_build_artifacts)
+
+    build_artifacts.add_argument(
+        'change',
+        help='number of the patch to run')
+
+    args = parser.parse_args()
+    args.command(args)
+
+
+def run_build_artifacts(args):
+    logging.basicConfig(
+        level=logging.INFO,
+        format="%(asctime)s %(levelname)-7s [%(name)s] %(message)s")
```

Comment:
Configuring the logger should be done in main(), not in the sub command. A sub command should only care about
the actual command. Everything else should be handled by main.

Reply:
I removed the logger, please let me know if you have different intentions.